style.css

```diff
@@ -310,3 +345,64 @@ body {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 708px) {
+  .work {
+    display: flex;
+    max-width: 70%;
+    margin: auto;
+    justify-content: space-between;
+  }
+
+  header {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0%;
+  }
+
+  .desktop {
+    margin-left: 50px;
+  }
+
+  .about-section {
+    display: flex;
+    width: 100%;
+  }
```

Comment:
- [ ] It would be nicer to add paddings to the left and the right of the about section to improve your UI design:
<img width="704" alt="image" src="https://user-images.githubusercontent.com/77190203/160141248-d26dd2af-d897-419d-ac3c-a513c624dfec.png">